README.md

```diff
@@ -1,178 +1,114 @@
-# composer-prototype
+# Composer (prototype)
+
 The prototype of BF composer, showing the architecure, and the extension system.
 
-# architecture
+# Architecture
 
 ## components
 
-This project mainly includes three components
-1. Composer app
-   
-    Composer app is a web app providing editing experience on the underlying bot. Composer will reference one bot in the `bots` folder. 
-
-    Composer app can also start the bot, by invoking `bot launcher`, then allow users to test bot in composer. 
-
-2. Bot launcher
-
-    Bot launcher will load an bot from `bots` folder, and start a bot instance, ready to serve. It will allow composer and anyone to connect to. 
-
-3. Bots
```

Comment:
Why is the /Bots directory a constraint? We should let someone give us a path to any bot from any path that they choose.

Reply:
it's not a constraint, it's defined in the "bot" section in both Launcher and Composer, here I say bots, i just want to make the point, that the actual storage of bot is decoupled from the processing of bot (by Launcher or Composer)

Reply:
ah, i see. ok. i dont think we need to reference it here in this document then